src/s3-resource-selector/s3-in-context/index.tsx

```diff
@@ -85,7 +85,12 @@ export const S3InContext = React.forwardRef(
     return (
       <div className={styles.root}>
         <div className={styles.layout}>
-          <InternalFormField className={styles['layout-uri']} label={i18nStrings?.inContextUriLabel} stretch={true}>
+          <InternalFormField
+            className={styles['layout-uri']}
+            label={i18nStrings?.inContextUriLabel}
+            stretch={true}
+            __inheritParentFormField={false}
```

Comment:
The `div[role="group"]` announces parent formfield label, no need to label the <input> again.

Reply:
This is very fishy. S3 selector is a typical form field, should behave like any native one

Reply:
It stops <Input> to inherit the label from grandparent fromfield. 
```
<Formfield aria-label="label of the s3 selector">
  <Formfield aria-label="label of the input">
    <Input/>
</Formfield></Formfield>
```
If enter the input from s3 internally, it will announce "label of the input" but not announce "label of the s3 selector" again.